web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
```

Comment:
> the exclamation point seems to be useless in this context, except to make it obvious that it's authorize is a method

According to [another answer](http://stackoverflow.com/a/612653/2370599) on that linked SO page:
> In Rails an exclamation point often means that the method will throw an exception on failure rather than failing silently.

I'm a fan of that convention, personally, as it will tell you that you need to have some sort of exception handling.